orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/ServerGroupCacheForceRefreshTask.groovy

```diff
@@ -154,26 +157,46 @@ class ServerGroupCacheForceRefreshTask extends AbstractCloudProviderAwareTask im
                                                   String cloudProvider,
                                                   StageData stageData,
                                                   Long startTime) {
+    // log.debug("Force cache refresh calling clouddriver with refresh type ${REFRESH_TYPE}")
     def pendingForceCacheUpdates = cacheStatusService.pendingForceCacheUpdates(cloudProvider, REFRESH_TYPE)
-    boolean isRecent = (startTime != null) ? pendingForceCacheUpdates.find { it.cacheTime >= startTime } : false
+    log.debug("Force cache refresh clouddriver response was: ${pendingForceCacheUpdates}.")
+    boolean pendingFound = pendingForceCacheUpdates.find { it.cacheTime >= startTime }
+    boolean isRecent = (startTime != null) ? (pendingFound != null ) ? pendingFound : false : false
```

Comment:
This is effectively doing the same as before, right? It's arguably a little harder to read due to the nested ternary and `null` check of a `boolean` type.

Reply:
Yes, this can be reverted to what it was.